src/components/Button/Button.style.js

```diff
@@ -0,0 +1,78 @@
+import styled, { css } from "styled-components";
+
+export const StyledButton = styled.button`
+  cursor: pointer;
+  font-family: inherit;
+  font-size: 1.6rem;
+  border-radius: 8px;
+  border: none;
+  text-transform: uppercase;
+  padding: 1rem 3rem;
+  min-width: 30%;
+  min-height: 5rem;
+  background: transparent;
+
+  ${({ type }) =>
+    type === "primary" &&
```

Comment:
i think it shouldn't by type, it should just be prop
so if user pass create button like `<Button primary secondary default>...</Button>` only primary styles should display, but I am not sure if it won't overcomplicate